web/impact/impact/tests/test_user_list_view.py

```diff
@@ -32,10 +32,25 @@ def test_get(self):
         user2 = UserContext().user
         with self.login(username=self.basic_user().username):
             url = reverse("user")
+            user_count = User.objects.count()
             response = self.client.get(url)
+            results = response.data["results"]
+            assert user_count == min(len(results), 10)
             emails = [result["email"] for result in response.data["results"]]
             assert user1.email in emails
             assert user2.email in emails
+            assert user_count == response.data["count"]
+
+    def test_get_with_limit(self):
+        UserContext().user
+        UserContext().user
```

Comment:
Are these two lines doing something that one `UserContext()` wouldn't do alone?

Reply:
presumably this is so that we have 2 users total but want to limit the api call to 1 for comparison

Reply:
Ah, ok, thanks.  The `.user` seems superfluous to me, and we could use `create_batch` instead, but it works.